pink/rondis/string/db_operations.cc

```diff
@@ -614,3 +614,173 @@ int rondb_get_rondb_key(const NdbDictionary::Table *tab,
     }
     return 0;
 }
+
+#define RONDB_INSERT 2
+#define RONDB_UPDATE 1
+#define REG0 0
+#define REG1 1
+#define REG2 2
+#define REG3 3
+#define REG4 4
+#define REG5 5
+#define REG6 6
+#define REG7 7
+#define LABEL0 0
+#define LABEL1 1
```

Comment:
Can you add a comment what the labels are for?

Reply:
Not sure how to give them a useful meaning, they are simply a label to jump to.